docs/scope-and-objectives.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Cluster API Scope and Objectives
+authors:
+  - "@vincepri"
+  - "@ncdc"
+  - "@detiber"
+reviewers:
+  - "@timothysc"
+  - "@justinsb"
+creation-date: 2019-04-08
+last-updated: 2019-04-08
+---
+
+# Cluster API Scope and Objectives
+
+This is a living document that is refined over time. It serves as guard rails for what is in and out of scope for Cluster API. As new ideas or designs take shape over time, this document can and should be updated.
+
+## Table of Contents
+
+* [Statement and Objectives](#cluster-api-statement-and-objectives)
+  * [Metadata](#metadata)
+  * [Table of Contents](#table-of-contents)
+  * [Summary](#summary)
+  * [Glossary](#glossary)
+  * [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non\-goals](#non-goals)
+  * [Requirements](#requirements)
+  * [Workstreams](#workstreams)
+
+## Summary
+
+We are building a set of Kubernetes cluster management APIs to enable common cluster lifecycle operations (create, scale, upgrade, destroy) across infrastructure providers.
+
+#### What is Cluster API?
+
+- An API definition for managing Kubernetes Clusters declaratively.
+- A reference implementation for the core logic.
+- An API definition for cluster lifecycle management action extension points.
+
+## Glossary
```

Comment:
I think we should capitalize all words in the glossary, to help indicate that we're referring to a proper noun.  E.g. "Infrastructure Provider".